src/scripts/quests/Quests.ts

```diff
@@ -305,7 +305,12 @@ class Quests implements Saveable {
                 if (ql) {
                     ql.state(questLine.state);
                     if (questLine.state == QuestLineState.started) {
-                        ql.resumeAt(questLine.quest, questLine.initial);
+                        ql.resumeAt(questLine.quest, questLine?.initial instanceof Number ? questLine.initial : 0);
```

Comment:
I don't think we need the `questLine?.initial` as `questLine` should definitely exist.
The following should also work if `initial` isn't a number:
```suggestion
                        ql.resumeAt(questLine.quest, +questLine.initial || 0);
```